http.js

```diff
@@ -27,7 +27,8 @@ function start (entry, opts) {
   assert.equal(typeof opts, 'object', 'bankai/http: opts should be type object')
 
   var quiet = !!opts.quiet
-  var compiler = bankai(entry, { reload: true })
+  opts.reload = true
+  var compiler = bankai(entry, opts)
```

Comment:
Could we do:

```js
Object.assign({ reload: true }, opts })
```

Reply:
oh yea, sure! is this ok, or do you want it inline?